index.js

```diff
@@ -178,9 +185,16 @@ async function analyzeHbsFile(content) {
       if (node.params.length === 0) return;
 
       let firstParam = node.params[0];
-      if (firstParam.type !== 'StringLiteral') return;
-
-      translationKeys.add(firstParam.value);
+      if (firstParam.type === 'StringLiteral') {
+        translationKeys.add(firstParam.value);
+      } else if (firstParam.type === 'SubExpression' && firstParam.path.original === 'if') {
+        let conditionResults = firstParam.params.slice(1);
+        conditionResults.forEach(conditionResult => {
```

Comment:
why did you use `forEach()` here? I'm wondering if it would be easier to understand if we looked at `firstParam.params[1]` and `firstParam.params[2]` directly

Reply:
Yeah fine with that, just didnt want to repeat myself, but will change if you think it reads better